### PR TITLE
Export li data for cds report gen

### PR DESCRIPTION
**Pull request by jzohrab**
Generate raw data files for use by CDS when compiling reports.

## Summary

This adds a tool to generate files that can then be consumed during report generation in CDS.

Sample tool run:

`node tools/gen-raw-files.js -o ../coronadatascraper/dist-raw-li --date 2020-04-28`

This generates files and puts them in the `dist-raw-li` folder in CDS.  There will be new commands added to CDS to consume those files in a PR in that project.

Sample calls are documented in the comments of the tools/gen-raw-files.js module.

## Needing review

* can only put a single URL in the CDS report object -- similar with the "type", etc.  But multiple URL crawls go into a single source, so I just picked the first one.

## Notes

* The tool script is long, but it's well broken up, and it will eventually disappear entirely when everything is moved over.

### CODE REVIEW

File: tests/integration/shared/sources/new-or-changed-sources-test.js [hunk 1]
```diff
@@ -87,9 +87,9 @@ test('Live crawl and scrape', async t => {
   for (const key of sourceKeys) {
     let crawlCompleted = false
     try {
-      await crawlSource({ source: key })
+      await { crawl }({ source: key })
```
Comment (jzohrab) [inline tests/integration/shared/sources/new-or-changed-sources-test.js:90]: That's funky, I didn't even know that kind of thing was possible. 👍

Comment (ryanblock) [reply tests/integration/shared/sources/new-or-changed-sources-test.js:90]: It's probably not, I might have fat-fingered it. :////

**Comment by jzohrab** [comment]
Force pushed update as I've merged in separate parts of this in other PRs.

**Comment by jzohrab** [comment]
Merging.  The only core code this changes is the move of `find-scraper.js` to lib, which is tested, and the generation of files can be changed independently.

**Pull request merged**